samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Shell.xaml.cs

```diff
@@ -277,4 +280,37 @@ private void NavigationViewControl_DisplayModeChanged(MUXC.NavigationView sender
 			}
 		}
 	}
+
+	public class MyLoadableSource : ILoadable
+	{
+
+		public event EventHandler? IsExecutingChanged;
+
+		private bool _isExecuting;
+		public bool IsExecuting
+		{
+			get => _isExecuting;
+			set
+			{
+				if (_isExecuting != value)
+				{
+					_isExecuting = value;
+					IsExecutingChanged?.Invoke(this, new());
+				}
+			}
+		}
+
+		public async Task Execute()
+		{
+			try
+			{
+				IsExecuting = true;
+				await Task.Delay(5000);
```

Comment:
maybe add a comment saying this delay is intentional?

Reply:
ah, i missed a bunch of debugging stuff that shouldn't be here, shouldn't be using this